bootcfg/storage/filestore.go

```diff
@@ -112,7 +112,12 @@ func (s *fileStore) ProfileList() ([]*storagepb.Profile, error) {
 	return profiles, nil
 }
 
-// IgnitionGet gets an Ignition Config template by name.
+// IgnitionPut creates or updates an Ignition template.
+func (s *fileStore) IgnitionPut(name string, config []byte) error {
+	return Dir(s.root).writeFile(filepath.Join("ignition", name), config)
```

Comment:
This is done for all `FileStore` writes using the `Dir`, only allows writes beneath the given root (i.e. inside the directory). Its based on `http.Dir`. This is a great thing to call out though, thanks!

https://github.com/coreos/coreos-baremetal/blob/master/bootcfg/storage/fs.go#L22
tests: https://github.com/coreos/coreos-baremetal/blob/master/bootcfg/storage/fs_test.go#L46